buildpipeline/Core-Setup-Publish.json

```diff
@@ -223,7 +223,7 @@
       "continueOnError": false,
       "alwaysRun": false,
       "displayName": "Publish",
-      "condition": "and(succeeded(), ne(variables.PB_PublishType, ''))",
+      "condition": "and(succeeded(), notin(variables.PB_PublishType, '', 'blob'))",
```

Comment:
I don't think comparing to specific values meshes with the idea of PB_PublishType being a kind of flags enum. I don't see a good way to handle this without breaking apart the steps or passing in PB_PublishType to msbuild, though.

Reply:
For now the idea is that these two publish steps will be mutually exclusive, until we can fully refactor the publishing. 

I agree that the full configurability with the flags is not yet supported but that requires more work before it can be done.